packages/video_player/video_player_avfoundation/ios/Classes/AVAssetTrackUtils.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <AVFoundation/AVFoundation.h>
+
+/**
+ * Note: https://stackoverflow.com/questions/64161544
+ * `AVAssetTrack.preferredTransform` can have wrong `tx` and `ty`.
```

Comment:
This sentence should be after the one below, not before it; the first sentence in a declaration comment for a function should say what the function does. Adding more details/context is for later sentences.